qa/L0_perf_analyzer_report/models/identity_onnx_cache_off/1/model.onnx

```diff
@@ -0,0 +1,15 @@
+backend-test:[
+
+xy"Identity
test_identityZ
+x
+
+
+
+
+b
+y
+
+
+
+
+B
```

Comment:
we usually generate models with `gen_qa_models`. Can we add this model and its configs there?